### PR TITLE
Code clean up in VIPCfg. Remove TranslationCacheManager member from VIPCfg because TranslationCacheManager is using Singleton design pattern.

### PR DESCRIPTION
**Pull request by jessiejuachon**

### CODE REVIEW

**Review by Xiaochao8 — DISMISSED** [review]

File: src/main/java/com/vmware/vipclient/i18n/base/cache/TranslationCacheManager.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
     private TranslationCacheManager() {
     }
 
-    public static synchronized TranslationCacheManager createTranslationCacheManager() {
+    public static synchronized TranslationCacheManager getTranslationCacheManager() {
```
Comment (huihuiw01) [inline src/main/java/com/vmware/vipclient/i18n/base/cache/TranslationCacheManager.java:31]: Is it better rename the method name as 'getInstance'?

Comment (jessiejuachon) [reply src/main/java/com/vmware/vipclient/i18n/base/cache/TranslationCacheManager.java]: Made this name change. Please approve, @Xiaochao8 and @huihuiw01 .Thanks!

**Review by Xiaochao8 — DISMISSED** [review]

File: src/main/java/com/vmware/vipclient/i18n/base/L10nTypeEnum.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.vmware.vipclient.i18n.base;
+
+import com.vmware.vipclient.i18n.VIPCfg;
+import com.vmware.vipclient.i18n.base.cache.FormattingCache;
+import com.vmware.vipclient.i18n.base.cache.MessageCache;
+
+public enum L10nTypeEnum {
```
Comment (huihuiw01) [inline src/main/java/com/vmware/vipclient/i18n/base/L10nTypeEnum.java:7]: Looks like this new added class is not used, so what's the purpose of adding it?

Comment (jessiejuachon) [reply src/main/java/com/vmware/vipclient/i18n/base/L10nTypeEnum.java:7]: Added this by mistake. Removing it now.

**Review by huihuiw01 — APPROVED** [review]

**Review by Xiaochao8 — APPROVED** [review]

**Pull request merged**